src/request/export/ExportWords.js

```diff
@@ -83,11 +83,23 @@ class ExportWords extends Nimiq.Observable {
                 this._goToRecoveryWords(this._key);
             }
         });
+
         this._wordsPasswordBox.on(PasswordBox.Events.SUBMIT, this._passwordSubmitted.bind(this));
-        $recoveryWordsContinue.addEventListener('click', () => {
-            this._validateWords.reset();
-            window.location.hash = ExportWords.Pages.VALIDATE_WORDS;
+
+        this.$recoveryWordsContinue.classList.add('disabled');
```

Comment:
Why use a css class on the button that doesn't affect the style but only serves as a flag?
Use a boolean flag instead.